romana/kubernetes/resources.go

```diff
@@ -198,26 +197,33 @@ func CreateDefaultPolicy(o KubeObject, l *kubeListener) {
 
 	var desiredAction networkPolicyAction
 
-	if isolation, ok := o.Metadata.Annotations["net.alpha.kubernetes.io/network-isolation"]; ok {
-		log.Printf("Handling default policy on a namespace %s, isolation is now %s \n", o.Metadata.Name, isolation)
-		switch isolation {
-		case "on":
-			desiredAction = networkPolicyActionDelete
-		case "off":
-			desiredAction = networkPolicyActionAdd
-		default:
-			log.Printf("In CreateDefaultPolicy :: Error :: unrecognised annotation on a namespace %s is %s (expected on|off) \n",
-				o.Metadata.Name, isolation)
+	if np, ok := o.Metadata.Annotations["net.beta.kubernetes.io/networkpolicy"]; ok {
+		log.Printf("Handling default policy on a namespace %s, policy is now %s \n", o.Metadata.Name, np)
+		policy := struct {
```

Comment:
It doesn't need to be anonymous, but it makes it easier to throw away when network policies get more complex it future kubernetes releases (eg: egress policies, more fields that need reacting to.)